ambry-tools/src/main/java/com/github/ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -214,11 +217,13 @@ static void updateResourceIdealState(String destZkString, String destClusterName
    */
   static IdealState buildIdealState(String resource, Set<String> partitionSet) {
     FullAutoModeISBuilder builder = new FullAutoModeISBuilder(resource);
-    builder.setStateModel(LeaderStandbySMD.name);
+    builder.setStateModel(OnlineOfflineSMD.name);
     for (String partition : partitionSet) {
       builder.add(partition);
     }
-    builder.setRebalanceStrategy(CrushEdRebalanceStrategy.class.getName());
+    builder.setMinActiveReplica(MIN_ACTIVE_REPLICAS);
+    builder.setRebalanceDelay((int) REBALANCE_DELAY);
+    builder.setRebalancerClass(DelayedAutoRebalancer.class.getName());
```

Comment:
Would it possible to store the ideal state as a json doc in config property and have the populate tool take it as an argument? It seems clunky to have to modify the code every time we want to tweak our Helix config.

I realize the HelixVcrStateModel does need to know the state names.

Reply:
done.